default.nix

```diff
@@ -75,5 +69,14 @@ let
                     --prefix PATH : "${pkgs.lib.makeBinPath iohk-ops-extra-runtime-deps}"
                   '';
                });
+
+    #mkDerivation = args: super.mkDerivation (args // {
+    #enableLibraryProfiling = false;
+    #});
+  };
+});
+  cabal2nixpkgs = rec {
```

Comment:
@domenkozar, it uses `iohk-ops-extra-runtime-deps` which uses `self.yaml`, to avoid pulling in a set of different haskell packages.